addons/armaos/functions/fnc_shell_findLoginUser.sqf

```diff
@@ -17,16 +17,25 @@ private _users = _computer getVariable ["AE3_Userlist", createHashMap];
 
 private _result = [];
 
-if (_username in _users) then 
+if (_username == "root") then
```

Comment:
Permits root access in general. 
To fix:
```
if (_username in _users) then
{...}else
{
   if (_username isEqualTo "root") then
   {...};
};
```

Reply:
I will change that.